lib/ziti-tunnel-cbs/include/ziti/ziti_tunnel_cbs.h

```diff
@@ -86,39 +87,39 @@ XX(identifier, string, none, id, __VA_ARGS__) \
 XX(dump_path, string, none, dump_path, __VA_ARGS__)
 
 #define TNL_ENABLE_MFA(XX, ...) \
-XX(identifier, string, none, id, __VA_ARGS__)
+XX(identifier, string, none, Identifier, __VA_ARGS__)
```

Comment:
Is there a reason to capitalize the json fields? It seems tedious to me.

Reply:
I see. You want to camel-case the fields that are compound words, which is better than using underscores.

Reply:
I made those field names same as what UI currently supports, WDE UI is using CamelCase